packages/commonwealth/client/scripts/views/pages/manage_community/helpers.ts

```diff
@@ -40,9 +40,13 @@ export const setChainCategories = async (
             },
           ]);
         } else if (!create) {
-          app.config.chainCategories = app.config.chainCategories.filter(
-            (data) => data.id !== response.result.id
-          );
+          // TODO: this is a hack, we should be able to just remove the category from the list
```

Comment:
Can you explain what this does / affects? I don't understand.

Reply:
There are times where there is no response returned and it is null causing the chainEntity routes to error. This only performs the chainCategory removal if the response is nonNull